wpilib/wpilib/adxl345_i2c.py

```diff
@@ -145,27 +144,17 @@ def getAccelerations(self):
                 rawData[1] * self.kGsPerLSB,
                 rawData[2] * self.kGsPerLSB)
 
-    # Live Window code, only does anything if live window is activated.
 
-    def getSmartDashboardType(self):
-        return "3AxisAccelerometer"
+    def _updateValues(self, entryX, entryY, entryZ):
+        data = self.getAccelerations()
+        entryX.setDouble(data[0])
+        entryY.setDouble(data[1])
+        entryZ.setDouble(data[2])
 
-    def initTable(self, subtable):
-        if subtable is not None:
-            self.xEntry = subtable.getEntry("X")
-            self.yEntry = subtable.getEntry("Y")
-            self.zEntry = subtable.getEntry("Z")
-            self.updateTable()
-        else:
-            self.xEntry = None
-            self.yEntry = None
-            self.zEntry = None
-
-    def updateTable(self):
-        if self.xEntry is not None:
-            self.xEntry.setDouble(self.getX())
-        if self.yEntry is not None:
-            self.yEntry.setDouble(self.getY())
-        if self.zEntry is not None:
-            self.zEntry.setDouble(self.getZ())
+    def initSendable(self, builder):
+        builder.setSmartDashboardType("3AxisAccelerometer")
+        entryX = builder.getEntry("X")
+        entryY = builder.getEntry("Y")
+        entryZ = builder.getEntry("Z")
 
+        builder.setUpdateTable(self._updateValues(entryX, entryY, entryZ))
```

Comment:
This doesn't seem like what you want. This would call `_updateValues`, which would then return None, so you're calling `builder.setUpdateTable(None)`. Did you mean to put that in a lambda?

Do we have a test for this? It seems odd to me that this would pass a decent test; if we do have a test already, maybe we need a more general and thorough test.